mi/dataset/parser/test/test_metbk_a_dcl.py

```diff
@@ -111,6 +115,7 @@ def test_big_giant_input(self):
         # In a single read, get all particles in this file.
         number_expected_results = RECORDS_FILE_0_863
         result = parser.get_records(number_expected_results)
+        particle_to_yml(result, os.path.join(RESOURCE_PATH, YML_0_863))
```

Comment:
if we run particle_to_yaml every time the tests are run, we circumvent regression checking

Reply:
Oh yes, I forgot to get rid of them. Thanks